doc/tutorial/network_ring.rst

```diff
@@ -0,0 +1,337 @@
+6. Define the **simulation**.
+7. **Run** the simulation.
+8. Collect and visualize the **results**.
+
+The cell
+********
+
+Step **(1)** shows how a simple cell with a dendrite is created. We construct the following :term:`morphology` and label the soma and dendrite:
+
+.. figure:: ../gen-images/tutorial_network_ring_morph.svg
+   :width: 400
+   :align: center
+
+   A 4-segment cell with a soma (pink) and a branched dendrite (light blue).
+
+.. code-block:: python
+
+   # (1) Build a segment tree
+   tree = arbor.segment_tree()
+
+   # Soma (tag=1) with radius 6 μm, modelled as cylinder of length 2*radius
+   s = tree.append(arbor.mnpos, arbor.mpoint(-12, 0, 0, 6), arbor.mpoint(0, 0, 0, 6), tag=1)
+
+   # Single dendrite (tag=3) of length 50 μm and radius 2 μm attached to soma.
+   b0 = tree.append(s, arbor.mpoint(0, 0, 0, 2), arbor.mpoint(50, 0, 0, 2), tag=3)
+
+   # Attach two dendrites (tag=3) of length 50 μm to the end of the first dendrite.
+   # Radius tapers from 2 to 0.5 μm over the length of the dendrite.
+   b1 = tree.append(b0, arbor.mpoint(50, 0, 0, 2), arbor.mpoint(50+50/sqrt(2), 50/sqrt(2), 0, 0.5), tag=3)
+   # Constant radius of 1 μm over the length of the dendrite.
+   b2 = tree.append(b0, arbor.mpoint(50, 0, 0, 1), arbor.mpoint(50+50/sqrt(2), -50/sqrt(2), 0, 1), tag=3)
+
+   # Associate labels to tags
+   labels = arbor.label_dict()
+   labels['soma'] = '(tag 1)'
+   labels['dend'] = '(tag 3)'
+
+In step **(2)** we create a :term:`label` for both the root, and the site where we'll be connecting the cell to another:
+
+.. figure:: ../gen-images/tutorial_network_ring_synapse_site.svg
+   :width: 400
+   :align: center
+
+   We'll create labels for the root (red) and a synapse_site (black).
+
+.. code-block:: python
+
+   # (2) Mark location for synapse at the midpoint of branch 1 (the first dendrite).
+   labels['synapse_site'] = '(location 1 0.5)'
+   # Mark the root of the tree.
+   labels['root'] = '(root)'
+
+Step **(3)** defines a basic cell decor and creates the :ref:`cable cell <cablecell>` :ref:`description <modelcelldesc>`. In the decor, a synapse with an exponential decay (``'expsyn'``) is placed on the ``'synapse_site'`` **(4)**. A spike detector is placed at the ``'root'``.
+
+.. code-block:: python
+
+   # (3) Create a decor and a cable_cell
+   decor = arbor.decor()
+
+   # Put hh dynamics on soma, and passive properties on the dendrites.
+   decor.paint('"soma"', 'hh')
+   decor.paint('"dend"', 'pas')
+
+   # (4) Attach a single synapse.
+   decor.place('"synapse_site"', 'expsyn')
+
+   # Attach a spike detector with threshold of -10 mV.
+   decor.place('"root"', arbor.spike_detector(-10))
+
+   cell = arbor.cable_cell(tree, labels, decor)
+
+The recipe
+**********
+
+Our objective is creating a network of :ref:`connected <interconnectivity>` cells, so we need to create a :class:`arbor.recipe` accommodating multiple cells. Besides multiple cells, the recipe is also where the connections between cells is defined.
```

Comment:
To create a model with multiple connected cells, we need to use a recipe that describes the model.
The recipe is where the different cells and the connections between them are defined.